examples/README.md

```diff
@@ -46,21 +44,21 @@ $ npm install
 
 When adding a new example, please follow these guidelines:
 
-* Add your code in a single JavaScript or TypeScript file per example, directly in the `examples` folder
-* Do not introduce other dependencies in your example
-* Give your `.js` file a meaningful name using `-` separators e.g. `adding-to-a-stream.js` / `adding-to-a-stream.ts`
-* Indent your code using 2 spaces
-* Use the single line `//` comment style and comment your code
-* Add a comment at the top of your `.js` / `.ts` file describing what your example does
-* Add a comment at the top of your `.js` / `.ts` file describing any Redis commands that need to be run to set up data for your example (try and keep this minimal)
-* Use semicolons
-* Use `async` and `await`
-* Use single quotes, `'hello'` not `"hello"`
-* Use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) when embedding expressions in strings
-* Unless your example requires a connection string, assume Redis is on the default localhost port 6379 with no password
-* Use meaningful example data, let's not use `foo`, `bar`, `baz` etc!
-* Leave an empty line at the end of your `.js` file
-* Update this `README.md` file to add your example to the table
+- Add your code in a single JavaScript or TypeScript file per example, directly in the `examples` folder
```

Comment:
@simonprickett It's my eslint and prettier that formatted that file on save. Sorry for trouble.

Reply:
@simonprickett Hey so sorry i just realised i changed the order of readme file's coding guidelines list. it happened because i copied that readme file from previous pull request and somehow changed the order. i hope you understand my concern. Again extremely sorry for changing unwanted things. (P.S. I am so excited that i am contributing into redis community so, given that state i might have rushed it.)

Reply:
Please don't rush @sailingwithsandeep and only change the things that are necessary :) please change it back and we will re-review.

Reply:
@simonprickett @SuzeShardlow Hey i committed the changes hopefully this time there won't be an issue. Again sorry for the trouble.